components/button/index.css

```diff
@@ -0,0 +1,139 @@
+.btn {
+	padding: 0.5rem 1rem;
+	border-radius: 0.25rem;
+	border: 1px solid #efefef;
+	font-size: 1rem;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	gap: 0.5rem;
+}
+
+.btn:hover {
+	cursor: pointer;
```

Comment:
Better to use this directly in  .btn class rather than here, You can see differences when you gonna use transition.